app/js/controllers/actions/incomes/incomes-list-ctrl.js

```diff
@@ -22,6 +22,7 @@ angular
 		}
 
 		_.each($scope.data, element => {
+			element = preProcess.convertTimeAtrtributes(element, Array("recievedOn"));
```

Comment:
since you are using `...attributeArray`, "recievedOn" doesn't need to be in an array. u could do `element = preProcess.convertTimeAtrtributes(element, "recievedOn")`, unless you want to change the signature of the function